### PR TITLE
1.2 release preparations

### PR DESCRIPTION
**Pull request by eile**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -105,12 +128,27 @@ to discard information. The following filters are provided:
...
+# Changes {#Changes}
+
+## Version 1.2 {#Changes12}
+
+* [38](https://github.com/Eyescale/hwsd/pull/38): OS X: only find main
+  display in hwsd_gpu_cgl module
+* [37](https://github.com/Eyescale/hwsd/pull/37): MSVC: Fix static init deadlock
+* [33](https://github.com/Eyescale/hwsd/pull/33): Port to Qt5
+
+## Version 1.1 {#Changes11}
```
Comment (tribal-tec) [inline README.md]: put date like in https://github.com/Eyescale/Collage/blob/master/doc/Changelog.md?

File: doc/RelNotes.md [hunk 2]
```diff
@@ -1,86 +1,3 @@
...
-
-HWSD uses CMake and [Buildyard](https://github.com/Eyescale/Buildyard)
-to create a platform-specific build environment. The following platforms
-and build environments are tested:
-
-* Linux: Ubuntu 14.04, RHEL 6.5 (Makefile, i386, x64)
-* Windows: 7 (Visual Studio 2008, i386, x64)
-* Mac OS X: 10.8 (Makefile, i386, x64)
 
 Technical questions can be posted to the eq-dev Mailing List, or
```
Comment (tribal-tec) [inline doc/RelNotes.md]: move this README and remove this file?

File: README.md (hunk 1 above)
Comment (eile) [inline README.md]: done.

File: doc/RelNotes.md (hunk 2 above)
Comment (eile) [inline doc/RelNotes.md]: done

**Pull request merged**